### PR TITLE
update security tools page

### LINKED ISSUE #988: V2 layout Security tools page UI.
update the security tools page to the updated design.

image

image

### PR DESCRIPTION
**Pull request by AdimekweEbuka**
Close #988 
This PR updates the security tools page to current design.

godcr-_testnet_-2022-06-25-15-23-44

### CODE REVIEW

**Review by devchoplife — CHANGES_REQUESTED** [review]
Clicking on the `Sign message` button does nothing, It should lead to the sign message page

**Comment by AdimekweEbuka** [comment]
> Clicking on the `Sign message` button does nothing, It should lead to the sign message page

Yes, I left that out because I was also working on the sign message page but I'll do that now.

File: ui/values/localizable/en.go [hunk 1]
```diff
@@ -301,6 +301,7 @@ unlockWithPassword = "Unlock with password"
 "setupMixerInfo" = "%v Two dedicated accounts %v mixed %v & %v unmixed %v will be created in order to use the mixer. %v This action cannot be undone.%v"
 "backupInfo" = "%v Please backup your seed words and keep them in a safe place in order to recover your funds if your device gets lost or broken. %v Anyone who has your seed words can spend your funds! Do not share them.%v"
 "signMessageInfo" = "%v Signing a message with an address' private key allows you to prove that you are the owner of a given address to a possible counterparty.%v"
+"securityToolsInfo" = "%v Various tools that help in different aspects of cryptocurrency security will be located here. %v"
```
Comment (Sirmorrison) [inline ui/values/localizable/en.go]: move this to the bottom so we know string that need other language translations.

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

File: ui/page/wallets/wallet_page.go [hunk 2]
```diff
@@ -1204,7 +1204,7 @@ func (pg *WalletPage) HandleUserInteractions() {
 			if menu.button.Clicked() {
 				switch menu.id {
 				case SignMessagePageID:
-					pg.ParentNavigator().Display(NewSignMessagePage(pg.Load, listItem.wal))
+					pg.ParentNavigator().Display(NewSignMessagePage(pg.Load))
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_page.go]: This option should be removed since it has been migrated to the security tools page.

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

File: ui/values/localizable/en.go [hunk 3]
```diff
@@ -463,6 +463,7 @@ unlockWithPassword = "Unlock with password"
 "destAddr" = "Destination Address"
 "myAcct" = "My account"
 "selectWalletToOpen" = "Select the wallet you would like to open."
+"securityToolsInfo" = "%v Various tools that help in different aspects of cryptocurrency security will be located here. %v"
```
Comment (Sirmorrison) [inline ui/values/localizable/en.go]:
```suggestion
"securityToolsInfo" = "%v Various tools for enhancing your fund security, are located here. %v"
```

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

File: ui/values/localizable/en.go [hunk 4]
```diff
@@ -463,6 +463,7 @@ unlockWithPassword = "Unlock with password"
 "destAddr" = "Destination Address"
 "myAcct" = "My account"
 "selectWalletToOpen" = "Select the wallet you would like to open."
+"securityToolsInfo" = "%v Various tools for enhancing your fund security are, located here. %v"
```
Comment (Sirmorrison) [inline ui/values/localizable/en.go]:
```suggestion
"securityToolsInfo" = "%v Various tools for enhancing your fund security, are located here. %v"
```

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

**Review by Sirmorrison — APPROVED** [review]

**Pull request merged**